src/Views/MainPage/components/Content/Input/Input.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import { withStyles } from '@material-ui/core';
+import TextField from '@material-ui/core/TextField';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import styles from './Input.style';
+
+class Input extends PureComponent {
+  state = {
+    error: false,
+  }
+
+  losesFocus = () => {
+    const { input: { isReady, touched } } = this.props;
+    if (touched === false && isReady === false) {
```

Comment:
`touched === false && isReady === false` to jest to samo co `!touched && !isReady`. Nie ma potrzeby przyrównywać do boole'owej wartości w takiej sytuacji